client/components/product-card/options.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import ProductCardPriceGroup from './price-group';
+
+const ProductCardOptions = props => {
+	const {
```

Comment:
Same as above for moving this one to the arg above so we'd destructure and assign at the same time.

Reply:
This has been addressed in the [latest commit](https://github.com/Automattic/wp-calypso/pull/36834/commits/c3fb1db60f042e857140c33eb869bc7ac00e84a9).